app/controllers/calls_controller.rb

```diff
@@ -1,7 +1,14 @@
 class CallsController < ApplicationController
   skip_before_action :verify_authenticity_token
-
-  def begin
+  @@user_number = ""
+  @@has_called = "false"
```

Comment:
It doesn't look like there's a reason for using a string here. This should be a boolean.